api/src/chat/services/conversation.service.ts

```diff
@@ -182,6 +206,13 @@ export class ConversationService extends BaseService<
           'Conversation Model : No conversation has been updated',
         );
       }
+
+      //TODO: add check if nothing changed don't update
+
+      this.subscriberService.updateOne(convo.sender, {
```

Comment:
```suggestion
      await this.subscriberService.updateOne(convo.sender, {
```

Reply:
Yes, I'm aware of that. Thank you for pointing it out

Reply:
Thank you for the suggestion. However, I think we don't need to await here, as making the function async allows the update to happen in the background. This way, we can continue processing other tasks without waiting for it to complete, which can improve performance, especially if the update isn't critical for the next steps

Reply:
We need a await I think cause the next logic may need the subscriber to be up-to-date.